pixi.toml

```diff
@@ -36,9 +36,9 @@ lint-cpp-files = "clang-format --dry-run"
 lint-rerun = "python scripts/lint.py"
 lint-rs-files = "rustfmt --check"
 lint-rs-all = "cargo fmt --check"
-lint-py-black = "black --check --config rerun_py/pyproject.toml"
+lint-py-fmt-check = "ruff format --check --config rerun_py/pyproject.toml"
 lint-py-blackdoc = "blackdoc --check"
-lint-py-mypy = "mypy --install-types --non-interactive --no-warn-unused-ignore"
+lint-py-mypy = "mypy --no-warn-unused-ignore"
```

Comment:
Any reason for this change? Without `--install-types --non-interactive` mypy complains about a bunch of things related to usage of `requests` in a fresh environment.

```
~/rerun$ rm -rf .pixi/
~/rerun$ pixi run lint-py-mypy
examples/python/signed_distance_fields/download_dataset.py:10: error: Library stubs not installed for "requests"  [import]
examples/python/raw_mesh/download_dataset.py:10: error: Library stubs not installed for "requests"  [import]
examples/python/objectron/download_dataset.py:9: error: Library stubs not installed for "requests"  [import]
examples/python/objectron/download_dataset.py:53: error: Returning Any from function declared to return "str"  [no-any-return]
examples/python/structure_from_motion/main.py:16: error: Library stubs not installed for "requests"  [import]
examples/python/rgbd/main.py:19: error: Library stubs not installed for "requests"  [import]
examples/python/dicom_mri/main.py:23: error: Library stubs not installed for "requests"  [import]
examples/python/segment_anything_model/main.py:30: error: Library stubs not installed for "requests"  [import]
examples/python/open_photogrammetry_format/main.py:21: error: Library stubs not installed for "requests"  [import]
examples/python/human_pose_tracking/main.py:17: error: Library stubs not installed for "requests"  [import]
examples/python/depth_guided_stable_diffusion/main.py:18: error: Library stubs not installed for "requests"  [import]
examples/python/controlnet/main.py:20: error: Library stubs not installed for "requests"  [import]
examples/python/controlnet/main.py:20: note: Hint: "python3 -m pip install types-requests"
examples/python/controlnet/main.py:20: note: (or run "mypy --install-types" to install all missing stub packages)
examples/python/controlnet/main.py:20: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
examples/python/face_tracking/main.py:17: error: Library stubs not installed for "requests"  [import]
examples/python/detect_and_track_objects/main.py:16: error: Library stubs not installed for "requests"  [import]
Found 14 errors in 13 files (checked 297 source files)
```

Reply:
I had mypy behaving weird and so I wanted to match what the CI was executing (`mypy --no-warn-unused-ignore` via `justfile py-lint`)

Reply:
Why doesn't the CI need `--install-types --non-interactive`? Would it be any downside adding it to the CI side?